### PR TITLE
Update xss.py

### PR DESCRIPTION
**Pull request by QQQkl**
A small confusion in xss.py. The 'xss_request_uri' in line 149 should be 'xss_request_url'.

### CODE REVIEW

**Comment by sagarpo** [comment]
The check is for URI based XSS. Example: http://localhost/?test=<XSS_PAYLOAD>. test param in URL is again hardcoded param name. 

It should be ```xss_request_uri``` in only line no 150. Thanks for letting us know.

File: modules/xss.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def xss_get_url(url,method,headers,body,scanid=None):
                 dbupdate.insert_record(attack_result)
                 result = True
 
-        xss_request_uri = req.api_request(parsed_url+'/?test='+payload,"GET",headers)             
+        xss_request_url = req.api_request(parsed_url+'/?test='+payload,"GET",headers)             
         if xss_request_url.text.find(decoded_payload) != -1:
             impact = check_xss_impact(xss_request_uri.headers)
```
Comment (jc89) [inline modules/xss.py]: should be here `xss_request_url` instead of `xss_request_uri`?

Comment (sagarpo) [reply modules/xss.py]: @jc89 It should be xss_request_uri in line no 150.

Comment (sagarpo) [reply modules/xss.py]: Closing this pull request. I have done a couple of changes in the module. Thanks!

**Pull request closed without merging**